pkg/client/client.go

```diff
@@ -456,6 +460,14 @@ func (c *ConmonClient) Version(
 			return fmt.Errorf("create request: %w", err)
 		}
 
+		metadata, err := c.metadataBytes(ctx)
```

Comment:
should we only do these if we've previously EnabledTracing? We could save it in the client

Reply:
Yes, making it a no-op when tracing is disabled.